src/ota_metadata/legacy2/metadata.py

```diff
@@ -262,12 +278,6 @@ def _prepare_ota_image_metadata(
             )
             symlink_save_fpath.unlink(missing_ok=True)
 
-            # ------ post parsing ------ #
-            ft_regular_orm.orm_create_index(
-                index_name="digest_index",
-                index_keys=("digest",),
-            )
-
         logger.info(
```

Comment:
With the new `orm_bootstrap_db` API, now the indexes can be defined at defining the ORM. See `FileTableRegularORM` for more details.